modules/covid_research/spec/fixtures/files/valid-update-submission.json

```diff
@@ -1,83 +1,34 @@
 {
-	"diagnosed":true,
-	"DIAGNOSED_DETAILS":{
-		"DIAGNOSED_DETAILS::SYMPTOMS_ONLY":true,
-		"DIAGNOSED_DETAILS::ANTIBODY_BLOOD_TEST":false,
-		"DIAGNOSED_DETAILS::NASAL_SWAB_TEST_POSITIVE":false,
-		"DIAGNOSED_DETAILS::DIFFERENT_METHOD":true
-	},
-	"closeContactPositive":"UNSURE",
-	"hospitalized":true,
-	"smokeOrVape":false,
-	"HEALTH_HISTORY":{
-		"HEALTH_HISTORY::ALLERGY_VACCINE":true,
-		"HEALTH_HISTORY::AUTOIMMUNE_DISEASE":true,
-		"HEALTH_HISTORY::CANCER":false,
-		"HEALTH_HISTORY::DIABETES":false,
-		"HEALTH_HISTORY::HEART_DISEASE":false,
-		"HEALTH_HISTORY::HIGH_BLOOD_PRESSURE":false,
-		"HEALTH_HISTORY::IMMUNOCOMPROMISED":false,
-		"HEALTH_HISTORY::KIDNEY_LIVER_DISEASE":false,
-		"HEALTH_HISTORY::LUNG_DISEASE":false,
-		"HEALTH_HISTORY::STROKE":false,
-		"HEALTH_HISTORY::ANOTHER_SERIOUS_CHRONIC_ILLNESS":false,
-		"HEALTH_HISTORY::NONE_OF_ABOVE":false
-	},
-	"EMPLOYMENT_STATUS":{
-		"EMPLOYMENT_STATUS::EMPLOYED_HOME":true,
-		"EMPLOYMENT_STATUS::EMPLOYED_OUTSIDE_OF_HOME":false,
-		"EMPLOYMENT_STATUS::FRONTLINE_WORKER":false,
-		"EMPLOYMENT_STATUS::FURLOUGHED_UNEMPLOYED":false,
-		"EMPLOYMENT_STATUS::RETIRED":false,
-		"EMPLOYMENT_STATUS::STUDENT":false,
-		"EMPLOYMENT_STATUS::NONE_OF_ABOVE":false
-	},
-	"TRANSPORTATION":{
-		"TRANSPORTATION::CAR":true,
-		"TRANSPORTATION::FREQUENT_AIR_TRAVEL":true,
-		"TRANSPORTATION::PUBLIC_TRANSPORT":false,
-		"TRANSPORTATION::WALK_BIKE":false,
-		"TRANSPORTATION::WORK_FROM_HOME":false,
-		"TRANSPORTATION::NONE_OF_ABOVE":false
-	},
-	"residentsInHome":"THREE_FIVE",
-	"closeContact":"ELEVEN_THIRTY",
-	"veteranFullName":{
-		"first":"xdsfsdfds",
-		"last":"sdfsdfdsffds"
-	},
-	"email":"sdfsdfs@sefsfd.com",
-	"phone":"3334445555",
-	"zipCode":"22334",
-	"veteranDateOfBirth":"1977-05-06",
-	"VETERAN":{
-		"VETERAN::VETERAN":true,
-		"VETERAN::ACTIVE_DUTY":false,
-		"VETERAN::NATIONAL_GUARD_RESERVES":true,
-		"VETERAN::VA_EMPLOYEE":false,
-		"VETERAN::FAMILY_MEMBER_CAREGIVER":false,
-		"VETERAN::VA_HEALTHCARE_CHAMPVA":false,
-		"VETERAN::NONE_OF_ABOVE":false
-	},
-	"GENDER":{
-		"GENDER::MALE":true,
-		"GENDER::FEMALE":false,
-		"GENDER::TRANSGENDER_MALE":false,
-		"GENDER::TRANSGENDER_FEMALE":false,
-		"GENDER::SELF_IDENTIFY":true,
-		"GENDER::NONE_OF_ABOVE":false
-	},
-	"GENDER_SELF_IDENTIFY_DETAILS":"This is a gender self description",
-	"RACE_ETHNICITY":{
-		"RACE_ETHNICITY::AMERICAN_INDIAN_ALASKA_NATIVE":false,
-		"RACE_ETHNICITY::ASIAN":false,
-		"RACE_ETHNICITY::BLACK_AFRICAN_AMERICAN":false,
-		"RACE_ETHNICITY::HISPANIC_LATINO_SPANISH_ORIGIN":false,
-		"RACE_ETHNICITY::HAWAIIAN_PACIFIC_ISLANDER":true,
-		"RACE_ETHNICITY::WHITE":true,
-		"RACE_ETHNICITY::OTHER_RACE_ETHNICITY":false,
-		"RACE_ETHNICITY::NONE_OF_ABOVE":false
-	},
-	"consentAgreementAccepted":true,
+	"diagnosed": true,
```

Comment:
i know this is just a test fixture, but have you tried to submit this structure to the genisis test endpoint? i recall having a little trouble getting json structure perfected because of weird character limits and whatnot on their end.

Reply:
@batemapf not yet.  I am hoping to get this deployed to Staging where we can manage availability with feature toggles and test the genisis test endpoint directly from Staging.  I have not dug up the test credentials to try testing manually with curl but can also do that. Schema structure is still a bit fluid which is why the `update` schema is still a local file and not pushed to vets-json-schema yet.